packages/lerna/src/commands/add-caching/index.ts

```diff
@@ -156,6 +159,22 @@ class AddCachingCommand extends Command {
 
     writeJsonFile(nxJsonPath, nxJson);
   }
+
+  private async configureGitIgnore(): Promise<void> {
+    try {
+      await execa("git", ["check-ignore", ".nx/cache"]);
+      // .nx/cache is already ignored - no need to update .gitignore
+    } catch (e) {
+      try {
+        await writeFile(joinPathFragments(workspaceRoot, ".gitignore"), "\n.nx/cache\n", { flag: "a+" });
```

Comment:
I’m not familiar with passing this a+ flag, I would have gone for appendFile, I’m on mobile and very late but assuming this makes this equivalent?

Reply:
Yes, this is equivalent to using `appendFile`. I went ahead and changed it to use `appendFile` since that's more readable.